### PR TITLE
zh-translation: docs/root/intro/arch_overview/operations/init.rst

### PR DESCRIPTION
**Pull request by scilla0531**
ref: https://github.com/cloudnativeto/envoy/issues/21

For an explanation of how to fill out the fields, please see the relevant section
in [PULL_REQUESTS.md](https://github.com/envoyproxy/envoy/blob/master/PULL_REQUESTS.md)

Commit Message:
Additional Description:
Risk Level:
Testing:
Docs Changes:
Release Notes:
[Optional Runtime guard:]
[Optional Fixes #Issue]
[Optional Deprecated:]

### CODE REVIEW

File: docs/root/intro/arch_overview/operations/init.rst [hunk 1]
```diff
@@ -1,33 +1,15 @@
...
-  processing new connections before the draining of the old process begins.
+* 启动期间，:ref:`集群管理器 <arch_overview_cluster_manager>` 会经过一个多阶段的初始化，首先初始化静态/DNS 集群，然后初始化预定义的 :ref:`EDS <arch_overview_dynamic_config_eds>` 集群。如果适用，它将初始化 :ref:`CDS <arch_overview_dynamic_config_cds>`，等待响应（或失败） :ref:`一段有限的时间 <envoy_v3_api_field_config.core.v3.ConfigSource.initial_fetch_timeout>` ， 并为 CDS 提供的集群执行相同主/次初始化。
+* 如果集群使用 :ref:`主动健康检查 <arch_overview_health_checking>`，Envoy 也会执行一次主动健康检查。
+* 集群管理器初始化完成后，:ref:`RDS <arch_overview_dynamic_config_rds>` 和 :ref:`LDS <arch_overview_dynamic_config_lds>` 将进行初始化（如果适用）。服务器等待 LDS/RDS 响应（或失败） :ref:`一段有限的时间 <envoy_v3_api_field_config.core.v3.ConfigSource.initial_fetch_timeout>` 后，服务器才开始接受连接。
+* 如果 LDS 本身返回需要 RDS 响应的监听器，则 Envoy 会进一步等待 :ref:`一段有限的时间 <envoy_v3_api_field_config.core.v3.ConfigSource.initial_fetch_timeout>`，直到接收到 RDS 响应（或失败）为止。请注意，这个过程会在后续每个通过 LDS 添加的监听器进行，称为 :ref:`监听器热身 <config_listeners_lds>`。
+* 在完成前面的所有步骤之后，监听器开始接受新的连接。该流程可确保在热重启期间，新进程完全能够在旧进程被驱逐之前接受并处理新连接。
+
+初始化的关键设计原则是，始终确保 Envoy 在 :ref:`一段有限时间 <envoy_v3_api_field_config.core.v3.ConfigSource.initial_fetch_timeout>` 内进行初始化，在管理服务器可用性的前提下，尽最大努力获得完整的xDS配置集。
```
Comment (vgbhfive) [inline docs/root/intro/arch_overview/operations/init.rst]: 在管理服务器可用性的前提下，尽最大努力获得完整的 xDS 配置集。

File: docs/root/intro/arch_overview/operations/init.rst [hunk 2]
```diff
@@ -1,33 +1,15 @@
...
-  processing new connections before the draining of the old process begins.
+* 启动期间，:ref:`集群管理器 <arch_overview_cluster_manager>` 会经过一个多阶段的初始化，首先初始化静态/DNS 集群，然后初始化预定义的 :ref:`EDS <arch_overview_dynamic_config_eds>` 集群。如果适用，它将初始化 :ref:`CDS <arch_overview_dynamic_config_cds>`，等待响应（或失败） :ref:`一段有限的时间 <envoy_v3_api_field_config.core.v3.ConfigSource.initial_fetch_timeout>` ， 并为 CDS 提供的集群执行相同主/次初始化。
+* 如果集群使用 :ref:`主动健康检查 <arch_overview_health_checking>`，Envoy 也会执行一次主动健康检查。
+* 集群管理器初始化完成后，:ref:`RDS <arch_overview_dynamic_config_rds>` 和 :ref:`LDS <arch_overview_dynamic_config_lds>` 将进行初始化（如果适用）。服务器等待 LDS/RDS 响应（或失败） :ref:`一段有限的时间 <envoy_v3_api_field_config.core.v3.ConfigSource.initial_fetch_timeout>` 后，服务器才开始接受连接。
+* 如果 LDS 本身返回需要 RDS 响应的监听器，则 Envoy 会进一步等待 :ref:`一段有限的时间 <envoy_v3_api_field_config.core.v3.ConfigSource.initial_fetch_timeout>`，直到接收到 RDS 响应（或失败）为止。请注意，这个过程会在后续每个通过 LDS 添加的监听器进行，称为 :ref:`监听器热身 <config_listeners_lds>`。
+* 在完成前面的所有步骤之后，监听器开始接受新的连接。该流程可确保在热重启期间，新进程完全能够在旧进程被驱逐之前接受并处理新连接。
+
+初始化的关键设计原则是，始终确保 Envoy 在 :ref:`一段有限时间 <envoy_v3_api_field_config.core.v3.ConfigSource.initial_fetch_timeout>` 内进行初始化，在管理服务器可用性的前提下，尽最大努力获得完整的 xDS 配置集。
```
Comment (majinghe) [inline docs/root/intro/arch_overview/operations/init.rst]:
```suggestion
初始化的关键设计原则是，始终确保 Envoy 在 :ref:`初始化获取超时 <envoy_v3_api_field_config.core.v3.ConfigSource.initial_fetch_timeout>` 时间段内完成初始化，并在管理服务器可用性的前提下，尽最大努力获得完整的 xDS 配置集。
```

File: docs/root/intro/arch_overview/operations/init.rst [hunk 3]
```diff
@@ -1,33 +1,15 @@
...
-  response (or failure) is received. Note that this process takes place on every future listener
-  addition via LDS and is known as :ref:`listener warming <config_listeners_lds>`.
-* After all of the previous steps have taken place, the listeners start accepting new connections.
-  This flow ensures that during hot restart the new process is fully capable of accepting and
-  processing new connections before the draining of the old process begins.
+* 启动期间，:ref:`集群管理器 <arch_overview_cluster_manager>` 会经过一个多阶段的初始化，首先初始化静态/DNS 集群，然后初始化预定义的 :ref:`EDS <arch_overview_dynamic_config_eds>` 集群。如果适用，它将初始化 :ref:`CDS <arch_overview_dynamic_config_cds>`，等待响应（或失败） :ref:`一段有限的时间 <envoy_v3_api_field_config.core.v3.ConfigSource.initial_fetch_timeout>` ， 并为 CDS 提供的集群执行相同主/次初始化。
+* 如果集群使用 :ref:`主动健康检查 <arch_overview_health_checking>`，Envoy 也会执行一次主动健康检查。
+* 集群管理器初始化完成后，:ref:`RDS <arch_overview_dynamic_config_rds>` 和 :ref:`LDS <arch_overview_dynamic_config_lds>` 将进行初始化（如果适用）。服务器等待 LDS/RDS 响应（或失败） :ref:`一段有限的时间 <envoy_v3_api_field_config.core.v3.ConfigSource.initial_fetch_timeout>` 后，服务器才开始接受连接。
```
Comment (majinghe) [inline docs/root/intro/arch_overview/operations/init.rst]:
```suggestion
* 集群管理器初始化完成后，:ref:`RDS <arch_overview_dynamic_config_rds>` 和 :ref:`LDS <arch_overview_dynamic_config_lds>` 将进行初始化（如果适用）。服务器将等待 :ref:`一段有限的时间 <envoy_v3_api_field_config.core.v3.ConfigSource.initial_fetch_timeout>` ，来接受至少一个 LDS/RDS 请求的响应（或者失败）后，服务器才开始接受连接。

```

**Comment by majinghe** [comment]
@scilla0531   添加了些comment。参考一下。

**Pull request merged**